src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.logo {
+  max-width: 40px;
+  max-height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 10px;
+  align-items: center;
```

Comment:
The align-items property has no effect in this context because its parent (.nav__item) is not a flex or grid container. Remove it or add 'display: flex' to the .nav__item if needed.